### PR TITLE
Exams by Vasylchenko Danyil IM-33

### PR DESCRIPTION
**Pull request by D-Vasylchenko**
done:
-password.js
-day.js
-count.js
-filter.js
-sum.js
-random.js
-sample.js
-split.js
-month.js
-shuffle.js

### CODE REVIEW

**Comment by Kiryaoo** [comment]
Done a nice work, agree with all commits

**Comment by nikama-s** [comment]
I have reviewed all commits and agree with them

**Comment by namanica** [comment]
good job has been done!

**Comment by DavidkJun** [comment]
I absolutely agree with the commits in this files.

**Comment by alomj** [comment]
you have done a great job!

File: Tasks/password.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-    Index = Math.floor(Math.random() * MAX);
-    key = key + alphabet[Index];
-  }
-  return key;
+'use strict';
+const GeneratePassword = (alphabet, length) => {
+    const maxPass = alphabet.length;
+    let key = '';
+    for (let i = 0; i < length; i++) {
+        const index = Math.floor(Math.random() * maxPass);
```
Comment (tshemsedinov) [inline Tasks/password.js:7]: Decompose extracting random func

**Pull request closed without merging**